.github/workflows/contracts.yml

```diff
@@ -7,8 +7,8 @@ on:
     branches:
```

Comment:
What if we rename the file to `contracts-v1.yml`?
cc: @michalinacienciala

Reply:
I think it can be expanded to other `*.yml` files as well like `dashboard-*.yml` to stay consistent with the naming convention. Besides, we can also rename the names of the workflows at line 1 in each file where applicable. 
Btw. there's a nice feature request in GitHub to make [workflows run from subfolders](https://github.community/t/feature-request-allow-workflow-configuration-in-sub-folders/16776)

Reply:
Renamed 5 files here: 12b4c4c

Regarding the dashboard code being in `solidity-v1` we have plans to move the dashboard to a new home https://github.com/threshold-network/token-dashboard